symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/DatafeedLoopV2.java

```diff
@@ -46,6 +47,13 @@ public class DatafeedLoopV2 extends AbstractAckIdEventLoop {
    */
   private static final int DATAFEED_TAG_MAX_LENGTH = 100;
 
+  /**
+   * Regex pattern of fanout feeds, e.g. "e766c498eece0d113f035270ad6c3c63_f_f8001".
+   * We must exclude potential datahose feeds when we are reusing a datafeed.
+   * Datahose feeds are in the format *_p_*, e.g. "d25098517ec62f1fc65cd111667a8386_p_be940".
+   */
+  private static final Pattern FANOUT_FEED_PATTERN = Pattern.compile("^[^\\s_]+_f_[^\\s_]+$");
```

Comment:
is this part of the contract from DF API? Can't they just change that whenever they want?